main.cpp

```diff
@@ -49,88 +50,89 @@ arm.extend();                           // 3)
 arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
+
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        car
+//  action 1:    the car emits smog
+car.emitsSmog();
+//  action 2:    the car restricts gas flow
+car.restrictsGasFlow();
+//  action 3:    the car utilizes fuel
+car.utilizeFuel();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        elevator
+//  action 1:    the elevator carries passengers
+elevator.carriesPassengers();
+//  action 2:    the elevator goes up
+elevator.goesUp();
+//  action 3:    the elevator goes down
+elevator.goesDown();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        arm
+//  action 1:    the arm flexes
+arm.flexes();
+//  action 2:    the arm rotates
+arm.rotates();
+//  action 3:    the arm controls motion
+arm.controlsMotion();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        lion
+//  action 1:    a lion roars
+lion.roars();
+//  action 2:    a lion eats
+lion.eats();
+//  action 3:    a lion protects its pride
+lion.protectsPride();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        light bulb
+//  action 1:    a light bulb illuminates
+lightBulb.illuminates();
+//  action 2:    a light bulb burns out
+lightBulb.burnsOut();
+//  action 3:    a light bulb gets hot
+lightbulb.getsHot();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        window
+//  action 1:    the window opens
+window.opens();
+//  action 2:    the window passes light
+window.passesLight();
+//  action 3:    the window adds ventilation
+window.addsVentilation();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        speaker
+//  action 1:    the speakers produces sound
+speaker.producesSound();
+//  action 2:    the speaker consumes energy
+speaker.consumesEnergy();
+//  action 3:    the speaker vibrates
+speaker.vibrates();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        piano    
+//  action 1:    the piano sustains a note
+piano.sustainsNote();
+//  action 2:    the piano dampens notes
+piano.dampensNote();
+//  action 3:    the piano resonates
+piano.resonates();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.